client/src/js/analyses/components/Create/Create.js

```diff
@@ -32,7 +32,7 @@ export const CreateAnalysis = ({
     hasHmm,
     sampleId,
     show,
-    subtractions,
+    subtractionList,
```

Comment:
This is not a very descriptive variable name. Let's do something like `allSubtractions` or come up with something better.